### DISCUSSION TITLE
agreement(s) to share your contact informations

### DISCUSSION THREAD

**Discussion by gmat** [pyannote/pyannote-audio#1368] [General]
Hi @hbredin ,
I'm planing to share a pipeline transcription script which will use yours pyannote models. Do I guest all the users to create a login on huggingface and share their contacts or do you prefer the use of a shared token ?
Many thanks for this work

**Comment by hbredin** [comment]
Not sure whether it is safe for you to share a token and whether Huggingface recommends doing that.

Also note that nothing prevents you from skipping the whole token thing by downloading the models and sharing them directly with your users. See https://github.com/pyannote/pyannote-audio/blob/develop/FAQ.md#can-i-use-gated-models-(and-pipelines)-offline for more details.

**Reply by hbredin** [reply]
As long as you follow the rules of the MIT license obviously :)
Both pyannote source code and (some) pretrained models are MIT.

**Comment by gmat** [comment]
OK Cool :+1: 
Thank you for your reply
